Formula/remake.rb

```diff
@@ -33,10 +33,10 @@ class Remake < Formula
   conflicts_with "make", because: "both install texinfo files for make"
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
     system "make", "install"
+    # Remove texinfo files for make to avoid conflict
+    info.glob("make.info*").map(&:unlink)
```

Comment:
Also done by:
- Arch - https://github.com/archlinux/svntogit-community/blob/packages/remake/trunk/PKGBUILD#L32
- Gentoo - https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-devel/remake/remake-4.3.1.1.5.ebuild#n39
- MacPorts - https://github.com/macports/macports-ports/blob/master/devel/remake/Portfile#L43

And similarly not packaged by:
- Debian - https://packages.debian.org/sid/amd64/remake/filelist
- Fedora - https://packages.fedoraproject.org/pkgs/remake/remake/fedora-rawhide.html#provides / https://packages.fedoraproject.org/pkgs/remake/remake-doc/fedora-rawhide.html#provides